### DISCUSSION TITLE
File is not Password protected

### DISCUSSION THREAD

**Discussion by MisterHurg** [illsk1lls/ZipRipper#48] [Q&A]
1st time user, so please excuse the ignorance. I ran the program as instructed, it ran its course on an item that has a password. But it states the item is not password protected. Any ideas?

**Comment by illsk1lls** [comment]
How large is the file?

**Comment by MisterHurg** [comment]
About 1.8 gig.

**Comment by illsk1lls** [comment]
I'm noticing over ~1.5 gig is being reported often as "not password protected", possibly due to hashing issues.. =/

I will look further into this

**Comment by kaktinjanka** [comment]
I have th same problem. Files are not larger than 2MB. I also made a test file with empty .txt file inside, same problem. Windows 10 OS.

**Reply by illsk1lls** [reply]
The issue may be resolved in the latest version. I recently added a missing zip hash type.
